<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveUI</class>
 <widget class="QWidget" name="CurveUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CurveTool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../../.designer/icon/tooliocn/CurveIcon.png</normaloff>../../../../../../../../.designer/icon/tooliocn/CurveIcon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="CurveToolUI" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="ContorlTool">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="createCurvetab">
         <attribute name="title">
          <string>控制器</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>创建曲线</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0,0" columnstretch="1,0,0,0">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="6" column="0" colspan="4">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
             </layout>
            </item>
            <item row="8" column="0">
             <widget class="QPushButton" name="IkBt">
              <property name="text">
               <string>IK</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QRadioButton" name="fuziRadBt">
              <property name="text">
               <string>父子关系</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QPushButton" name="IKFKBt">
              <property name="text">
               <string>IKFK</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>修改颜色</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>曲线大小</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="EndRadBt">
              <property name="text">
               <string>endObj</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QRadioButton" name="lianjieRadBt">
              <property name="text">
               <string>链接</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>替换曲线</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <layout class="QVBoxLayout" name="verticalLayout_5"/>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QPushButton" name="replaceCurveBt">
              <property name="text">
               <string>替换曲线</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QDoubleSpinBox" name="CurveScaleNum">
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>20.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="8" column="3">
             <widget class="QPushButton" name="FKcijiBt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>FK带次级</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QPushButton" name="FKbt">
              <property name="text">
               <string>FK</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QRadioButton" name="yueshuRadBt">
              <property name="text">
               <string>约束</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="loadCurveBt">
              <property name="text">
               <string>载入</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="CurveName"/>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="CurveScaleBt">
              <property name="text">
               <string>缩放</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="4">
             <widget class="QPushButton" name="MirrorCurveBt">
              <property name="text">
               <string>镜像曲线</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="curveTool">
         <attribute name="title">
          <string>曲线设置</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0" colspan="3">
           <widget class="QPushButton" name="create_curveLocBT">
            <property name="text">
             <string>曲线控制点（loc）</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="loadCurveBt02">
            <property name="text">
             <string>载入</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="CurveSelectLE"/>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QPushButton" name="extracCurvetBt">
            <property name="text">
             <string>提取</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>提取曲线</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../guyupiTools/icon/iconClass/iocn.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
